File: bronze/5597(과제 안 내신 분..?)/5597.c
#include <stdio.h>
int main(){
    int n;
    int student[30]={};
    for (int i=0;i<28;i++){
        scanf("%d",&n);
        student[n-1]=1;
    }
    for (int i=0;i<30;i++){
        if (student[i]==0){
            printf("%d\n",i+1);
        }
    }
    return 0;
}

File: bronze/5597(과제 안 내신 분..?)/5597.java
import java.util.*;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int n;
        int student[] = new int[30];
        for (int i=0;i<28;i++){
            n = in.nextInt();
            student[n-1] = 1;
        }
        for (int i=0;i<30;i++){
            if (student[i] == 0){
                System.out.println(i+1);
            }
        }
        in.close();
        return;
    }
}
